docs/families.docs.js

```diff
@@ -0,0 +1,232 @@
+const familyEndpoints = {
+  '/families': {
+    post: {
+      tags: ['families'],
+      summary: 'Created object families in response',
+      description: 'families router',
+      operationId: 'postLoggedFamily',
+      produces: ['application/json'],
+      security: {
+        type: "basic",
+      },
+      parameters: [
+        {
+          in: 'body',
+          name: 'families',
+          description: 'Created object family',
+          required: true,
+          type: 'string',
+          schema: {
+            $ref: '#/definitions/FamilyCreatedRequest',
+          },
+        },
+        {
+          in: 'header',
+          name: 'Authorization',
+          description: 'Authorization',
+          required: true,
+          type: 'string',
+        }
+      ],
+      responses: {
+        '400': {
+          description: 'Request body format is invalid',
+        },
+        '401': {
+          description: 'User not authorized',
+        },
+        '409': {
+          description: 'User already created family/is a part of family',
+        },
+        '200': {
+          description: 'Family was created in DB successfully',
+          schema: {
+            $ref: '#/definitions/Family',
+          },
+        },
+      },
+    },
+
+    put: {
+      tags: ['families'],
+      summary: 'Update info from family document',
+      description: 'Update family',
+      operationId: 'putLoggedFamily',
+      produces: ['application/json'],
+      parameters: [
+        {
+          in: 'body',
+          name: 'Update object family',
+          description: 'Family update',
+          required: true,
+          type: 'string',
+          schema: {
+            $ref: '#/definitions/FamilyUpdateRequest',
+          },
+        },
+        {
+          in: 'header',
+          name: 'Authorization',
+          description: 'Authorization',
+          required: true,
+          type: 'string',
+        }
+      ],
+      responses: {
+        '400': {
+          description: 'User request body format is invalid',
+        },
+        '401': {
+          description: 'User not authorized',
+        },
+        '409': {
+          description: 'User already created family / is a part of family',
+        },
+        '200': {
+          description: 'Family was created in DB successfully',
+          schema: {
+            $ref: '#/definitions/Family',
+          },
+        },
+      },
+      security: {
```

Comment:
we don't need security option here or anywhere else, since we're passing directly Authorization hearer